addon/templates/components/bourbon-accordion-item.hbs

```diff
@@ -0,0 +1,12 @@
+{{#if listItem.header}}
```

Comment:
So, as written, if I defined `header` and `headerPartial`, both would be displayed! Intended?

Reply:
yeah.  that was intended to give flexibility.  i guess i can make the component smarter to only take one or the other.  thoughts?

Reply:
In practice, I'm not sure people would make the mistake? So I'm on the fence.